src/bun.js/api/bun/socket.zig

```diff
@@ -224,7 +224,9 @@ pub const SocketConfig = struct {
 
             const port_value = opts.get(globalObject, "port") orelse JSValue.zero;
             if (port_value.isEmptyOrUndefinedOrNull() or !port_value.isNumber() or port_value.toInt64() > std.math.maxInt(u16) or port_value.toInt64() < 0) {
-                exception.* = JSC.toInvalidArguments("Expected \"port\" to be a number between 0 and 65432", .{}, globalObject).asObjectRef();
+                exception.* = JSC.toInvalidArguments("Expected \"port\" to be a number between 0 and {d}", .{
```

Comment:
we can do `comptimePrint` but actually just hardcode it

Reply:
Done in https://github.com/oven-sh/bun/pull/1705/commits/3fdcdbe8f0bb5408e7e7e3a4b5cb9c21613a123f